src/markdown/parsers/cardParser.ts

```diff
@@ -135,12 +135,16 @@ export class CardParser extends BaseParser {
    * @private
    */
   async linesToHtml(lines: string[]) {
+    const fixLatex = (match: string) => (
+      match.replace(/\\[{}%#&$_\\]/g, str => str === "\\\\" ? "\\\\\\\\" : "\\" + str)
+    );
     const string = lines.join("\n")
       // $$\{1,2\} \%100$$ => $$\\{1,2\\} \\%100$$
-      .replace(/(?<!\\)\$\$\n?(.+?)\n?(?<!\\)\$\$/g, str => str.replace(/(\\[{}%#&$_])/g, "\\$1"))
+      .replace(/(?<!\\)\$\$\n?.+?\n?(?<!\\)\$\$/g, fixLatex)
       // $\{1,2\} \%100$ => $\\{1,2\\} \\%100$
-      .replace(/(?<![\\$])\$(?!\$)(.+?)(?<!\\)\$/g, str => str.replace(/(\\[{}%#&$_])/g, "\\$1"));
+      .replace(/(?<![\\$])\$(?!\$).+?(?<!\\)\$/g, fixLatex);
 
+    console.log(string);
```

Comment:
You may want to remove this

Do we need tests for this?, It would be great if you were able to add some as I'm not sure what's being fixed here..

Reply:
it has been removed.